proto/import_kvpb.proto

```diff
@@ -37,7 +37,7 @@ service ImportKV {
     // Open an engine.
     rpc OpenEngine(OpenEngineRequest) returns (OpenEngineResponse) {}
     // Open a write stream to the engine.
-    rpc WriteEngine(stream WriteEngineRequest) returns (WriteEngineResponse) {}
```

Comment:
Could you keep this old method? This allows backward compatibility.

Reply:
Oh Sure. New name indicates keeping backward compatibility, which I just realized :P